docs/source/core_services/service_agents/emailer/EmailerAgent.rst

```diff
@@ -0,0 +1,29 @@
+.. _EmailerAgent:
+
+===============
+Emailer Agent
+===============
+Emailer agent is responsible for sending emails for an instance. It has been written so that any agent on the instance
+can send emails through it via the "send_email" method or through the pubsub message bus using the topic
+"platform/send_email".
+
+By default any alerts will be sent through this agent. In addition all emails will be published to the
+"record/sent_email" topic for a historian to be able to capture that data.
+
+Configuration
+~~~~~~~~~~~~~
+A typical configuration for this agent is as follows. We need to specify the SMTP server address or the mail gateway,
+email address of the sender, email addresses of all the recipients and frequency (in minutes) with which the emails need
```

Comment:
the SMTP server is the mail gateway so use SMTP server and get rid of mail gateway in the sentance.